src/models/searchservice/response/DirectAnswer.ts

```diff
@@ -15,7 +15,7 @@ export interface DirectAnswer {
    * @remarks
    * A value will not be present if the {@link DirectAnswer.fieldType} is 'rich_text'.
    */
-  value?: string,
+  value?: T,
```

Comment:
I know it was like this before, but why is value optional?

Reply:
IIRC, when we get a rich text field value, the value attribute is not set.